### ISSUE TITLE
[Diagram][connectors] Add context menu

### ISSUE THREAD

**Issue by umlsynco** [umlsynco/umlsync-framework#30]
Add context menus for the connectors:
1. Remove
2. Add Label

**Comment by umlsynco** [comment]
- Removed Framework dependency from the Diagrammer Module
- Added connector's context menu but via stubs only (added context.isConnector flag for the trigger)

**Comment by umlsynco** [comment]
[ ] Implement connector remove functionality 
[ ] Implement add label + coordinates; Depends on #29 
[ ] Dedicated context menu for the labels ["remove", "edit"]

**Comment by umlsynco** [comment]
- Dedicated context menu for the labels ["remove", "edit"] moved to the separate task #32 
Because it takes too much changes
